### PR TITLE
VUE-1389: overall cleanup of saved-search interface [VUE-1389] [VUE-1390] [VUE-1391]

### PR DESCRIPTION
**Pull request by tiarascoleman**
Screen Shot 2022-11-14 at 9 32 50 AM
Screen Shot 2022-11-14 at 9 33 22 AM

Included screenshots of desktop and mobile interfaces

### CODE REVIEW

File: src/components/Lend/LoanSearch/LoanSearchInterface.vue [hunk 1]
```diff
@@ -60,22 +69,22 @@
...
 						@updated="handleUpdatedFilters"
 						@reset="handleResetFilters"
 					/>
-					<p class="tw-hidden lg:tw-block tw-mt-1">
+					<p class="tw-hidden lg:tw-inline-block tw-mt-1 tw-align-center">
 						{{ totalCount }} Loans
 					</p>
+					<loan-search-saved-search
+						class="tw-hidden lg:tw-inline-block tw-mt-1 tw-align-center tw-justify-self-auto"
+						v-if="enableSavedSearch && showSavedSearch && !savedSearchSuccess"
```
Comment (mcstover) [inline src/components/Lend/LoanSearch/LoanSearchInterface.vue:83]: `!savedSearchSuccess` can be removed from both instances here and above as it's now been removed and won't be defined.

**Review by mcstover — APPROVED** [review]
This is the bulk of the work. A little more polish on the interface but it's good to go ahead and get this PR merged in

**Review by dyersituations — APPROVED** [review]
@mcstover We'll probably need to center align the UI vertically, but approving so it can be merged, and that can be solved later.

File: src/components/Lend/LoanSearch/LoanSearchInterface.vue (hunk 1 above)
Comment (tiarascoleman) [reply src/components/Lend/LoanSearch/LoanSearchInterface.vue]: Fixed!

**Pull request merged**